src/EnergyPlus/UnitarySystem.cc

```diff
@@ -3858,7 +3789,7 @@ namespace UnitarySystems {
             this->ControlZoneNum = UtilityRoutines::FindItemInList(loc_controlZoneName, state.dataHeatBal->Zone);
         } else if (this->m_ControlType == UnitarySysCtrlType::Load || this->m_ControlType == UnitarySysCtrlType::CCMASHRAE) {
             if (this->m_sysType < SysType::PackagedAC) {
```

Comment:
Doing `<`, `>` comparisons on enumerations is not good practice. Better (from both a robustness standpoint and a readability standpoint) to do a bunch of `==` comparisons.

Reply:
I cleaned up all of these.